dashboard/config/application.rb

```diff
@@ -1,4 +1,3 @@
-require File.expand_path('../deployment', __FILE__)
```

Comment:
I believe this is referencing the deleted 'deployment.rb' file.

Reply:
I believe this needs to be replaced with
```suggestion
require File.expand_path('../../../deployment', __FILE__)
```
to fix the issue with loading cdo/poste on the next line

Reply:
Yep, i was just coming to that conclusion myself, thanks!